packages/client/src/v2-events/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { ResolvedUser } from '@opencrvs/commons'
+
+/**
+ *  Human readable date format for full date and time (e.g. January 09, 2025 · 01.24 PM)
+ */
+export const HUMAN_READABLE_FULL_DATE_TIME = 'MMMM dd, yyyy · hh.mm a'
```

Comment:
Let's bring this in from localisation items right away. I think there might even be an item for this already or if no, we can create it

Reply:
All date formatting needs to be configurable by countries. The mechanism for that are the localisation files, so `client.csv` for instance where in this case you could have 

```
"events.recordView.creationTime": "MMMM dd, yyyy · hh.mm a"
```

and a 

```ts
{
  id: 'events.recordView.creationTime',
  defaultMessage: 'MMMM dd, yyyy · hh.mm a'
}
```

somewhere else in the code